src/logging.cpp

```diff
@@ -36,9 +36,21 @@ quill::Logger* file_io_logger;  // for file io
 
 const static std::filesystem::path LOG_FILE = log_dir() / "app.log";
 
+inline quill::Logger*
```

Comment:
How can this be inline in a CPP file. it worked fine in the header file. move it back